dataflow_config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "reserve_threshold": 2199023255552,
```

Comment:
The only reason I kept this in bytes is because python's shutil library returns values in bytes. This makes computation easier since all numbers stay in the same units.

Reply:
My comment is more with respect to end-user usability: having the units in the variable name makes things explicitly clear and means they don't have to scan through code comments. I understand the shutil units being in bytes, but the config value can just be multiplied by `1 << 20`.